magefiles/utils.go

```diff
@@ -17,6 +17,13 @@ func binaryWithExt(name string) string {
 	return name
 }
 
+func checkOs() string {
+	if runtime.GOOS == "windows" {
```

Comment:
Why can't you just return `runtime.GOOS` for windows?

Reply:
Thanks for pointing that out. That ```if``` condition was un-necessary.